INSTALL.md

```diff
@@ -1,246 +1,187 @@
+* tinyxml (Zlib): http://www.grinninglizard.com/tinyxml
+* zlib (Zlib): https://www.zlib.net
 
-## Windows dependencies
+### Linux dependencies
 
-Run
+To install the required dependencies on Debian/Ubuntu, run:
+
+    sudo apt-get install cmake g++ libfreetype6-dev libgif-dev libgtest-dev libjpeg-dev libpixman-1-dev libpng-dev libtinyxml-dev libx11-dev libxcursor-dev ninja-build zlib1g-dev
+
+### Windows dependencies
+
+To install the required dependencies with vcpkg, run:
 
     vcpkg install freetype giflib gtest libjpeg-turbo libpng libwebp pixman tinyxml zlib --triplet x64-windows
     
- Beware: `--triplet x64-windows` is only necessary for a 64 architecture
- build.
+Beware: `--triplet x64-windows` is only necessary for a 64 architecture
+build.
 
-After that you have to choose the back-end:
+### MacOS dependencies
 
-1. If you choose the Allegro back-end, you can jump directly to the
-   [Compiling](#compiling) section.
+On MacOS you will need Mac OS X 10.11 SDK and Xcode 7.3 (older versions
+might work).
 
-2. If you choose the Skia back-end, you will need to
-   [compile Skia](#skia-on-windows) before and then continue in the
-   [Compiling](#compiling) section. Remember to check the
-   [Windows details](#windows-details) section to know how to call
-   `cmake` correctly.
+## Compiling
 
-## Mac OS X dependencies
+To compile LibreSprite, run the following commands:
 
-On OS X you will need Mac OS X 10.11 SDK and Xcode 7.3 (maybe older
-versions might work).
+    cd LibreSprite
+    mkdir build
+    cd build
 
-Also, you must compile [Skia](#skia-on-mac-os-x) before starting with
-the [compilation](#compiling).
+Then following the platform-specific instructions for compiling below.
 
-## Linux dependencies
+After you've finished compiling, the executable file will be placed in the
+`build/bin` directory. If you want to build a fresh copy of LibreSprite, remove
+the `build` directory and recompile.
 
-Debian/Ubuntu:
+Once you've finished compiling, you can install LibreSprite with the following
+command:
 
-    sudo apt-get install cmake g++ libfreetype6-dev libgif-dev libgtest-dev libjpeg-dev libpixman-1-dev libpng-dev libtinyxml-dev libx11-dev libxcursor-dev ninja-build zlib1g-dev
+    ninja install
 
-The `libxcursor-dev` package is needed to
-[hide the hardware cursor](https://github.com/aseprite/aseprite/issues/913).
+### Linux details
 
-# Compiling
+To compile LibreSprite, run the following commands:
 
-1. [Get LibreSprite code](#get-the-source-code), put it in a folder like
-   `C:\LibreSprite`, and create a `build` directory inside to leave all
-   the files that are result of the compilation process (`.exe`,
-   `.lib`, `.obj`, `.a`, `.o`, etc).
+    cmake -G Ninja ..
+	ninja libresprite
 
-        cd C:\LibreSprite
-        mkdir build
+The repository contains a patched version of the Allegro 4 library.
+If you want to use your installed version of Allegro, run `cmake` with
+the flag `-DUSE_SHARED_ALLEGRO4=ON`.
```

Comment:
"[...] `-DUSE_SHARED_ALLEGRO4=ON`, but beware, as it's highly unrecommended (1) (2).", where 1 and 2 contains links to the issues